lib/runctl.js

```diff
@@ -150,9 +154,15 @@ function onRequest(req, callback) {
     cluster.disconnect();
 
   } else if (cmd === 'fork') {
-    var child = cluster.fork();
-    rsp.workerID = child.workerID;
-    rsp.processID = child.process.pid;
+    rsp = null;
+    cluster.fork().once('fork', function(worker) {
```

Comment:
I thought this was a pure-refactor, and you just didn't like assigning to rsp, now I understand better...

Please add a comment: the fork event is emitted *after* strong-cluster-control has extended it with some custom properties, which is why we can't simply use the fork() return value.